core/dbt/node_runners.py

```diff
@@ -118,7 +119,8 @@ def from_run_result(self, result, start_time, timing_info):
             skip=result.skip,
             status=result.status,
             failed=result.failed,
-            timing_info=timing_info
+            timing_info=timing_info,
+            agate_table=result.agate_table,
```

Comment:
As of python 3.something, this trailing comma just before a close parenthesis is legal syntax. I'm really excited to start using this for writing out multi-line keyword arguments, just like in dict/list definitions. So many fewer syntax errors in my future!